exercises/C-comparison-operators/exercise.js

```diff
@@ -7,14 +7,14 @@
 
 let studentCount = 16;
 let mentorCount = 9;
-let moreStudentsThanMentors; // finish this statement
+let moreStudentsThanMentors = studentCount > mentorCount; // finish this statement
 
 let roomMaxCapacity = 25;
-let enoughSpaceInRoom; // finish this statement
+let enoughSpaceInRoom = moreStudentsThanMentors <= roomMaxCapacity; // finish this statement
```

Comment:
**moreStudentsThanMentors** is a Boolean value. Do you want to find out the total number of students and mentors and then check if the room is big enough?